### PR TITLE
Add face mask question

### PR DESCRIPTION
**Pull request by the-kenneth**
# Description

Adds face mask question. Develops further on @MikeD-ZOE 's vitamin question (nicely done to encapsulate the creation of the dto in the object) by using an enum to clean up the typing. 

Needs translations and a full test, but is basically there.

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

### CODE REVIEW

File: assets/lang/en.json [hunk 1]
```diff
@@ -851,5 +851,21 @@
     "sign-off": "Thank you for helping our scientists to understand COVID-19 better, and provide analysis to the NHS daily.",
     "cta-single-profile": "Back to home screen",
     "cta-multi-profile": "Report for others"
+  },
+
+  "face-mask": {
+    "worn-a-mask-question": "In the last week, did you wear a face mask when outside the house?",
+    "worn-a-mask-justification": "Why do we ask this: This information can help us understand if face masks reduce COVID infection risk. This question is completely optional.",
```
Comment (abubuwe) [inline assets/lang/en.json:858]:
```suggestion
    "worn-a-mask-justification": "This information can help us understand if face masks reduce COVID infection risk. This question is completely optional.",
```

**Comment by abubuwe** [comment]
Attached Swedish translations - Spanish inbound
```"face-mask": {
    "worn-a-mask-question": "Har du under den senaste veckan en ansiktsmask när du har lämnat ditt hem?",
    "worn-a-mask-justification": "Denna information kan hjälpa oss förstå om ansiktsmasker minskar risken för covid-19-infektion. Det är helt frivilligt att svara på denna fråga.",
    "never": "Aldrig",
    "sometimes": "Ibland",
    "most-of-the-time": "För det mesta",
    "always": "Alltid",
    "not-applicable": "Inte tillämpligt",
    "type-of-mask-question": "Vilken typ av ansiktsmask använder du? Markera alla tillämpliga",
    "cloth_scarf": "Tyg eller scarf",
    "surgical": "Kirurgisk mask",
    "respirator": "N95-/FFP-skyddsmask",
    "not-sure": "Inte säker/föredrar att inte berätta",
    "other": "Annan, ange vilken:"
  }

File: src/features/assessment/fields/FaceMaskQuestion.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      worn_face_mask: wornFaceMask,
+      mask_cloth_or_scarf: false,
+      mask_surgical: false,
+      mask_n95_ffp: false,
+      mask_not_sure_pfnts: false,
+      mask_other: otherMask,
+    } as Partial<AssessmentInfosRequest>;
+
+    selectedMasks.forEach((selectedMask) => {
+      if (selectedMask != TypeOfMaskValues.OTHER) {
```
Comment (selvach) [inline src/features/assessment/fields/FaceMaskQuestion.tsx:89]: !=== here

File: src/features/assessment/fields/FaceMaskQuestion.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    { label: i18n.t('face-mask.other'), value: TypeOfMaskValues.OTHER },
+  ];
+
+  render() {
+    const { formikProps } = this.props;
+    return (
+      <FieldWrapper>
+        <Item stackedLabel style={styles.textItemStyle}>
+          <Label>{i18n.t('face-mask.worn-a-mask-question')}</Label>
+          <Label style={styles.infoText}>{i18n.t('face-mask.worn-a-mask-justification')}</Label>
```
Comment (selvach) [inline src/features/assessment/fields/FaceMaskQuestion.tsx:119]: ☁️  - Not for this PR - but creating a reusable component for these InfoText / explanations might be useful.

**Pull request merged**